dev-client/src/components/sites/CreateSiteView.tsx

```diff
@@ -191,7 +191,9 @@ export const CreateSiteView = ({
 
   const onSave = useCallback(
     async (form: FormState) => {
```

Comment:
nit: this would be a bit tidier, but all good either way!
```suggestion
    async ({locationSource: _, ...form}: FormState) => {
```